### PR TITLE
Pull request to merge the features branch to the main

### PR DESCRIPTION
**Pull request by Micky373**
Hello reviewer,

I have finished sections 1 and 2 of the mobile portfolio skeleton. Check for any amendments and please let me know if there should be any amendments.

Thank you

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   <link href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic" rel="stylesheet" />
+   <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+   <link href="https://fonts.googleapis.com/css?family=Crete+Round:regular,italic" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/e971436820.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <header class="main-section">
+        <section class="toolbar">
+            <h3>John Doe</h3>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that your logo has a link (`a` tag)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            </div>
+            <ul class="socials">
+                <li><a href=""><i class="fab fa-github"></i></a></li>
+                <li><a href=""><i class="fab fa-linkedin-in"></i></a></li>
+                <li><a href=""><i class="fab fa-angellist"></i></a></li>
+                <li><a href=""><i class="fab fa-twitter"></i></a></li>
+                <li><a href=""><i class="fab fa-medium-m"></i></a></li>
+            </ul>
+        </section>
+    </header>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider, that you have to create two sections: toolbar (header) and headline section. In `body` tag should be two sections. Considering your code, you insert all inside of `header` tag. Please, add your toolbar (first section) into `header` tag and your 2nd headline section (which should be right after the header) into `section` tag, aka you should use HTML5 semantic tags.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+   <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+   <link href="https://fonts.googleapis.com/css?family=Crete+Round:regular,italic" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/e971436820.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <header class="main-section">
+        <section class="toolbar">
+            <h3>John Doe</h3>
+            <i class="fa fa-bars" aria-hidden="true"></i>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that your hamburger button has a `nav` tag

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Micky373 ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [ ] Please, consider comments under review

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by FrederickMih — APPROVED** [review]
### Hi @Micky373 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**